### PR TITLE
Set default date on component and selected event

### PR DESCRIPTION
**Pull request by DaniPoot**

### CODE REVIEW

File: src/pages/transactions/components/transaction-view.vue [hunk 1]
```diff
@@ -62,7 +62,7 @@ q-card.q-pa-sm.full-width
             v-model="transaction.value.memo"
             color="secondary"
         )
-    .col-2
+    .col-2.myCommit
```
Comment (jmgayosso) [inline src/pages/transactions/components/transaction-view.vue]: Do we using this class??

File: src/pages/transactions/components/transaction-view.vue [hunk 2]
```diff
@@ -537,10 +538,13 @@ export default {
       if (!this.addingComponent) {
         this.addingComponent = true
         const tempHash = [...Array(8)].map(() => Math.floor(Math.random() * 16).toString(16)).join('')
-        let d = new Date()
-        let month = d.getUTCMonth() + 1
-        let day = d.getUTCDate()
-        let date = `${d.getUTCFullYear()}/${(month < 10 ? '0' + month : month)}/${(day < 10 ? '0' + day : day)}`
+        let date = this.transaction.value.date
```
Comment (jmgayosso) [inline src/pages/transactions/components/transaction-view.vue:541]: Did we validating the transaction does not have a Date before set new one?

**Review by jmgayosso — COMMENTED** [review]
- Are we validating the transaction does not have a Date before set a new one?

File: src/pages/transactions/components/transaction-view.vue (hunk 1 above)
Comment (DaniPoot) [reply src/pages/transactions/components/transaction-view.vue]: I removed in the next commit

File: src/pages/transactions/components/transaction-view.vue (hunk 2 above)
Comment (DaniPoot) [reply src/pages/transactions/components/transaction-view.vue:541]: In the next line validate if the date variable is undefined or null I set a new one

**Review by jmgayosso — APPROVED** [review]
Looking good

**Pull request merged**